src/rgw/rgw_op.cc

```diff
@@ -872,6 +886,15 @@ int RGWGetObj::handle_user_manifest(const char *prefix)
   if (r < 0)
     return r;
 
+  if (!total_len) {
+    /* There were no non-zero segments in the DLO.
+     * So we still need to return status.
+     * (BZ #1380196)
```

Comment:
Can we please refer to a tracker.ceph.com ticket here, instead of Red Hat's Bugzilla?